chatlab/conversation.py

```diff
@@ -14,69 +14,28 @@
 import asyncio
 import logging
 import os
-from dataclasses import dataclass
-from typing import Callable, Dict, Iterable, List, Optional, Tuple, Type, Union, cast, overload
+from typing import Callable, List, Optional, Tuple, Type, Union, overload
 
 import openai
-import openai.error
 from deprecation import deprecated
 from IPython.core.async_helpers import get_asyncio_loop
+from openai import AsyncOpenAI
+from openai._streaming import AsyncStream
```

Comment:
Instead of pulling this inner type out, perhaps I can use the return type from `client.chat.completions.create`